client/my-sites/sidebar/sidebar.jsx

```diff
@@ -129,11 +129,13 @@ module.exports = React.createClass( {
 		}
 
 		return (
-			<li className={ this.itemLinkClass( '/ads', 'ads' ) }>
-				<a onClick={ this.onNavigate } href={ adsLink }>
-					<span className="menu-link-text">{ site.jetpack ? 'AdControl' : 'WordAds' }</span>
-				</a>
-			</li>
+			<SidebarMenuItem
+				label={ site.jetpack ? 'AdControl' : 'WordAds' }
```

Comment:
@ebinnion I think it's because it's a one word Proper Noun or Trademark or somesuch. I had it translated before and was told to un-translate it :)